templates/ts-apollo-fullstack/server/src/db.ts

```diff
@@ -17,3 +17,11 @@ export const createDB = async () => {
     const db = knex(generateConfig.dbmigrations)
     return db
 }
+
+export const createMongoDB = async () => {
+    const MongoClient = require('mongodb').MongoClient;
+    const url = "mongodb url";
```

Comment:
Is this a valid URL?

Reply:
I will remove that now as we will use knex by default